libtenzir/builtins/operators/sort.cpp

```diff
@@ -504,6 +506,62 @@ class plugin2 final : public virtual operator_plugin2<sort_operator2> {
                            make_sort_key);
     return std::make_unique<sort_operator2>(std::move(sort_exprs));
   }
+
+  auto make_function(method_plugin::invocation inv, session ctx) const
+    -> failure_or<function_ptr> override {
+    auto expr = ast::expression{};
+    TRY(argument_parser2::method(name()).add(expr, "<expr>").parse(inv, ctx));
```

Comment:
Note that we could consider adding a `recurse=true` function for recursive sorting for records, but I think this can wait until someone asks for it.